pkg/opensearch/lucene_handler.go

```diff
@@ -79,6 +91,18 @@ func (h *luceneHandler) processQuery(q *Query) error {
 	return nil
 }
 
+func getTraceId(rawQuery string) string {
+	trimmed := strings.TrimSpace(rawQuery)
+	re := regexp.MustCompile(`traceId:(.+)`)
+	matches := re.FindStringSubmatch(trimmed)
```

Comment:
What happens if something like `foo traceId:test` is passed? Does it get parsed successfully, and do we want that to happen?

Reply:
Before and after the refactor we've had it pass successfully and take into account just the trace id - we don't really support any additional filtering in the query field. Im ok with leaving it live that and not changing the functionality